README.md

```diff
@@ -3,60 +3,70 @@
 Galasa provides Helm charts to install various components, the main one being a Galasa Ecosystem.
 
 ## Usage
+**Note: The Galasa Ecosystem chart only supports x86-64 at the moment. It cannot be installed on ARM64-based systems.**
```

Comment:
I wouldn't recommend using an x86 VM on Arm64 since performance is poor, which leads to timeout errors and constant pod restarts. We can update this once a viable alternative is found and tested.